staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher_test.go

```diff
@@ -29,6 +29,7 @@ import (
 	"k8s.io/apimachinery/pkg/runtime/schema"
 	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
 	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/apimachinery/pkg/watch"
```

Comment:
Please also squash the commits - having multiple commits doesn't really help in this case (we generally make them in cases where just merging few of them would also make sense or for very clear readbility purposes like splitting autogenerated ones).

Reply:
squashed.